### PR TITLE
CA-246335: Only allow miami-era VM operations during rolling upgrade …

### PR DESCRIPTION
**Pull request by jonludlam**
…mode

We had code to limit the contents of allowed_operations, but nothing to
actually prevent the operations being performed, so a CLI user could do
anything.

Signed-off-by: Jon Ludlam <jonathan.ludlam@citrix.com>

### CODE REVIEW

**Comment by gaborigloi** [comment]
We could also add some unit tests at some point to `ocaml/xapi/test_vm_operations.ml`, which tests exactly this function :smile: .

**Comment by jonludlam** [comment]
funny you should say that about tests... was just doing exactly that :-)

File: ocaml/xapi/xapi_globs.ml [hunk 1]
```diff
@@ -423,7 +423,7 @@ let host_operations_miami = [
   `provision;
 ]
 
-let vm_operations_miami = [
+let rpu_allowed_vm_operations = [
```
Comment (robhoes) [inline ocaml/xapi/xapi_globs.ml]: You've not renamed the users yet?

Comment (jonludlam) [inline ocaml/xapi/xapi_globs.ml]: Oops, git add screwup :-)

**Pull request merged**